### PR TITLE
added documentation for the functional tests

### PR DESCRIPTION
**Pull request by fieranmason**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,23 @@ mem-admin has been built using MongoDB, Express, AngularJS and NodeJS.  See http
...
+* Spin up automated testing of the functional test instance of the application
+
+Note that you need the following env vars:
+* FUNCTIONAL_PORT=3001 -- used to server the functional test instance of the application
+* FUNCTIONAL_HOST=localhost -- location of the functional test instance of the application
+* BASEURL=http://localhost:3001 -- the url targeted by the functional tests
+* MONGODB_FUNC_HOST=localhost -- the host for the mongo instance targeted by the functional test instance of the application
+* MONGODB_FUNC_PORT=27017 -- port for the mongo instance targeted by the functional test instance of the application
+* MONGODB_FUNC_DATABASE=mem-dev-func -- name of the database used by the functional test instance of the application
+
```
Comment (NickPhura) [inline README.md]: Can we add to the prerequisites that an instance of Mongo needs to be running.

Comment (esune) [reply README.md]: +1 to Nick's comment.

File: README.md [hunk 2]
```diff
@@ -25,6 +25,25 @@ mem-admin has been built using MongoDB, Express, AngularJS and NodeJS.  See http
...
```
Comment (fieranmason) [inline README.md]: Added a comment to that effect

**Comment by fieranmason** [comment]
patched an issue with functional.js where it was by default pointing at mem-dev database, now it points at mem-dev func

**Pull request closed without merging**